### ISSUE TITLE
Use args over constants

### ISSUE THREAD

**Issue by wdsrocha** [wesleysr/optimization#3]
On `genetic_algorithms/Main.cpp` there are the following constants:

```c++
#define MAX_ITER 2
#define POP_SIZE 1000
#define MUT_RATE 0.07
#define ELIT_RATE 0.15
#define MAX_GEN 200
```
It would be interesting if they could be passed as arguments when executing the program.

If the arguments are emitted, use the existent constants.

**Comment by ayham-1** [comment]
Hey @wdsrocha , I would like to take this!

**Comment by wdsrocha** [comment]
Hi @realaltffour , it has been solved already (see #7), but thanks!
